### PR TITLE
multi_queues_test.with_netperf.cpu_affinity: modify the method of irq static to support  ppc64

### PR DESCRIPTION
**Pull request by Zhengtong**
id: 1314636

### CODE REVIEW

File: generic/tests/multi_queues_test.py [hunk 1]
```diff
@@ -64,9 +64,12 @@ def get_cpu_irq_statistics(session, irq_number, cpu_id=None):
...
         Get guest interrupts statistics
         """
+        online_cpu_number_cmd = r"cat /proc/interrupts | head -n 1 | wc -w"
         cmd = r"cat /proc/interrupts | sed -n '/^\s\+%s:/p'" % irq_number
+        online_cpu_number = session.cmd_output(online_cpu_number_cmd)
         irq_statics = session.cmd_output(cmd)
-        irq_statics_list = map(int, irq_statics.split()[1:-2])
+        irq_statics_list = map(int,
+                               irq_statics.split()[1:int(online_cpu_number)])
         if irq_statics_list:
```
Comment (PandaWei) [inline generic/tests/multi_queues_test.py]: looks good, 
do you have do regression test?

Comment (Zhengtong) [inline generic/tests/multi_queues_test.py]: Have done some tests, passed

File: generic/tests/multi_queues_test.py [hunk 2]
```diff
@@ -64,9 +64,12 @@ def get_cpu_irq_statistics(session, irq_number, cpu_id=None):
         """
         Get guest interrupts statistics
         """
+        online_cpu_number_cmd = r"cat /proc/interrupts | head -n 1 | wc -w"
         cmd = r"cat /proc/interrupts | sed -n '/^\s\+%s:/p'" % irq_number
+        online_cpu_number = session.cmd_output(online_cpu_number_cmd)
```
Comment (xutian) [inline generic/tests/multi_queues_test.py]: why not covert type of 'online_cpu_number' here but do it in  line#72

Comment (xutian) [inline generic/tests/multi_queues_test.py]: And suggest to use cmd_output_safe() at here.

**Comment by Zhengtong** [comment]
updated, thanks

**Pull request merged**